java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -212,4 +212,44 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<String> configSet(Map<String, String> parameters, Route route);
+
+    /**
+     * Returns the server time. <br>
+     * The command will be routed to a random node.
+     *
+     * @see <a href="https://redis.io/commands/time/">redis.io</a> for details.
+     * @return The current server time as a <code>String[]</code> with two elements: A Unix timestamp
+     *     and the amount of microseconds already elapsed in the current second. The returned array is
+     *     in a [Unix timestamp, Microseconds already elapsed] format.
+     * @example
+     *     <pre>{@code
+     * String[] serverTime = client.time().get();
+     * System.out.println("Server time is: " + serverTime[0] + "." + serverTime[1]);
+     * }</pre>
+     */
+    CompletableFuture<String[]> time();
+
+    /**
+     * Returns the server time.
+     *
+     * @see <a href="https://redis.io/commands/time/">redis.io</a> for details.
+     * @param route Specifies the routing configuration for the command. The client will route the
+     *     command to the nodes defined by <code>route</code>.
+     * @return The current server time as a <code>String[]</code> with two elements: A Unix timestamp
+     *     and the amount of microseconds already elapsed in the current second. The returned array is
+     *     in a [Unix timestamp, Microseconds already elapsed] format.
+     * @example
+     *     <pre>{@code
+     * String[] serverTime = client.time().get(RANDOM).getSingleValue();
```

Comment:
Probably will have to explain what random does with a comment and same goes for ALL_NODES. 
```
// Command sent to all nodes via ALL_NODES route. Expects MultiValue result.
// Command sent to a single random node via RANDOM route. Expects SingleValue result.
```